modAionBase/src/org/aion/base/util/ByteArrayWrapper.java

```diff
@@ -61,4 +61,32 @@ public ByteArrayWrapper fromBytes(byte[] bs) {
     public static ByteArrayWrapper wrap(byte[] data) {
         return new ByteArrayWrapper(data);
     }
+
+    /**
+     * Checks if every byte in the array has the value 0.
+     *
+     * @return {@code true} if every byte in the array has the value 0, {@code false} otherwise
+     */
+    public boolean isZero() {
+        int length = data.length;
+        for (int i = 0; i < length; i++) {
+            if (data[length - 1 - i] != 0) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
this method might need to benchmark to see this implementation is faster than pure byte compares by order.